packages/teleport-plugin-react-css-modules/src/index.ts

```diff
@@ -66,7 +66,7 @@ export const createPlugin: ComponentPluginFactory<ReactCSSModulesConfig> = (conf
 
     traverseElements(uidl.node, (element) => {
       const { style, key } = element
-      if (style) {
+      if (style && Object.keys(style).length > 0) {
```

Comment:
can you check how I solved this in the stencil-css plugin which is still experimental? I think it's better to check both for inline styles as well as for the external styles:

https://github.com/teleporthq/teleport-code-generators/pull/308/files#diff-499a2b85fe4af0f3722b2a33b09ccad4R47

without this, if you have a dynamic style with css-modules, the class is still generated

Reply:
Sure